bookshelf.js

```diff
@@ -0,0 +1,62 @@
+
+
+var button = document.createElement('button');
+button.setAttribute("id", "fetch-books-btn");
+var textNode = document.createTextNode('Fetch Books');
+button.appendChild(textNode);
+//button.innerHTML("Fetch Books");
+button.addEventListener("click",fetchBooks);
+
+var bookDoc = document.getElementsByTagName("body")[0];
+    bookDoc.appendChild(button);
+
+
+function fetchBooks(){
+    const booksJSON = 'https://raw.githubusercontent.com/codeyourfuture/bookshelf-project/master/books.json'
+    fetch(booksJSON)
+        .then(response => response.json())
+        .then(function processBooks(booksJSON){
+            //var bookArray = Array.from(booksJSON);
+            var mainListNode = document.createElement('ul')
+            button.addEventListener("click",inlineFunc);
```

Comment:
Not sure this is quite correct. The spec says to add an event listener to the unordered list item (`<ul>`) but this is adding the listener to the "Fetch books" button.